presto-docs/src/main/sphinx/release/release-0.226.rst

```diff
@@ -0,0 +1,21 @@
+=============
+Release 0.226
+=============
+
+General Changes
+_______________
+* Add peak running task count to statistics field of QueryCompletedEvent.
+* Fix ST_Buffer for tiny geometries (#13194)
```

Comment:
I think proper syntax for linking to issues is 
```
:issue:`13194`
```

Could you double check by building release notes locally and update throughout?